src/test/java/RenameAPlaylist.java

```diff
@@ -1,53 +1,66 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.Keys;
 
 public class RenameAPlaylist extends BaseTest {
 
-    String email = "holostenco.yuliya@gmail.com";
-    String Password = "te$t$tudent";
-    String xpath = "//ul//li[@class='playlist playlist']";
-    String css ="li>input";
-   String  newPlaylistName=" -Best Club music Selection";
+
+    By newPlaylistLocator = By.cssSelector(".playlist:nth-child(3)");
+    By getNewPlaylistTextFieldLocator =By.cssSelector("input[name='name']");
+    String  newPlaylistName= "Best";
```

Comment:
I recommend:  make all the locators private and variable newPlaylistName , because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
  private By newPlaylistLocator = By.cssSelector(".playlist:nth-child(3)");
  private By getNewPlaylistTextFieldLocator = By.cssSelector("input[name='name']");
  private String newPlaylistName = "Best";
```
![image](https://user-images.githubusercontent.com/104761482/200170443-c937898a-7b9a-4237-957a-d4140959fd32.png)